### ISSUE TITLE
Trilogy version support

### ISSUE THREAD

**Issue by ShawnSigma** [randovania/open-prime-rando#158]
### _Hey, not sure if this is the right place to ask. If it isn't, I'm sorry >~>_

I was wondering if this randomiser will ever support the Metroid Prime Trilogy versions of Metroid Prime 1 and 2 (mainly 2). It might sound weird at first, but it makes sense if you consider that the Wii version can be used in PrimeHack to play these games in what I think is the definitive way: mouse and keyboard, free look, higher FOV.
There's already a tool that can convert Metroid Prime 1 randomised ISOs into Trilogy ISOs, so it must be possible to do for Prime 2. I know it's a very subjective thing and the first response will probably something along the lines of "Grow up you big baby, learn to play with the original controls" but it's so bad compared to PrimeHack that it makes me wonder how that's not the default way to play these games in the community. 
Also, the developer of the tool I mentioned has already stated that they won't add Prime 2 support

### **TL;DR**
So yeah, I guess my suggestion is to work on Trilogy support (or ISO conversion, like the existing tool for Prime 1)
